src/main/java/ca/corefacility/bioinformatics/irida/plugin/amrdetection/StarAMRUpdater.java

```diff
@@ -0,0 +1,194 @@
+package ca.corefacility.bioinformatics.irida.plugin.staramr;
```

Comment:
Similar comment as above for this file. Could you move from the directory `amrdetection` to the directory `staramr`.